test/e2e/node/pod_resize.go

```diff
@@ -824,9 +884,18 @@ var _ = ginkgo.Describe("[sig-node] PodInPlaceResize", func() {
 			verifyPodContainersCgroupConfig(pod, tc.containers)
 
 			ginkgo.By("patching pod for resize")
-			pPod, pErr := f.ClientSet.CoreV1().Pods(pod.Namespace).Patch(pod.Name,
-				types.StrategicMergePatchType, []byte(tc.patchString))
-			framework.ExpectNoError(pErr, "failed to patch pod for resize")
+			if vmPod {
```

Comment:
where is the packing string defined ? for VMs, memory plug/unplug is not dynamic for this release, i.e. it is 512 MiB each. so the patch should keep this in mind. it will be round to this value.

Reply:
It's part of the test case. The runtime should be able to handle arbitrary memory requests & limits that pass apiserver validation.